### ISSUE TITLE
pod5 convert fast5 hangs indefinitely at 100% done 

### ISSUE THREAD

**Issue by wilsonte-umich** [nanoporetech/pod5-file-format#29]
I had no trouble getting `pod5 convert fast5` up and running by installing using pip into a conda environment on my Linux server running Slurm. Initial tests on smaller data sets/number of files worked fine. However, when I run the command on my full ONT dataset, the program gets to 100% and never exits.

```
# command
pod5 convert fast5 --threads 16 ./fast5_pass/*.fast5 --output pod5 --one-to-one fast5_pass

# hang state in log - stays here for hours
Converting 674 Fast5s: 100%|##########| 2695235/2695235 [2:27:51<00:00, 303.81Reads/s] 

# idle state of pod process
top
   PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND 
336777 wilsonte  20   0 9780.7m 223624  43464 S   0.0   0.1  18:18.87 pod5   

# verification of done state in file output
ls fast5_pass | wc -w
674
ls pod5 | wc -w
674
```
I can find no evidence that the command has anything more to do, or is doing anything, but it never exits, which prevents my pipeline from progressing. I have forced a stop and just continued on with Dorado basecalling - so far that seems to have no problems with the pod5 files created above.

**Comment by jorj1988** [comment]
Hi @wilsonte-umich ,

Interesting, we will investigate further and try to work out whats going on...

- George

**Comment by HalfPhoton** [comment]
Thanks for reporting this issue.
We've isolated this bug and are issuing a fix in `0.1.14` imminently.

**Comment by HalfPhoton** [comment]
This issue should now be resolved in the latest `0.1.15` release.

Thanks again for reporting this issue.

Best regards,
Rich
